lib/resources/r4/slot_patch.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: Slot
+field_name_base_url: https://hl7.org/fhir/r4/slot-definitions.html#Slot
+operations:
+  - name: replace-status
+    path: /status
+    operation: replace
+    type: code
+    url: https://hl7.org/fhir/R4/slot-definitions.html#Slot.status
+    description: The status of the slot.
+    example: |
+      {
+        "op": "replace",
+        "path": "/status"
+        "status": "free"
+      }
+    note: >
+      <ul>
+        <li>Only `free` and `busy tentative` are supported.</li>
```

Comment:
do we want 'busy-tentative' here? or was this on purpose

Reply:
It should have the “-“

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/pull/608/commits/ea23b86cb0ea922efebe9793e03bc621b6cd6906)